include/olt-manager.php

```diff
@@ -4,13 +4,14 @@
  */
 
 /**
- * It is best practice that plugins do nothing before plugins_loaded is fired
- * so it is what Polylang intends to do
- * but some plugins load their text domain as soon as loaded, thus before plugins_loaded is fired
- * this class differs text domain loading until the language is defined
- * either in a plugins_loaded action or in a wp action ( when the language is set from content on frontend )
+ * It is best practice that plugins do nothing before `plugins_loaded` is fired.
+ * So it is what Polylang intends to do.
+ * But some plugins load their textdomain as soon as loaded, thus before `plugins_loaded` is fired.
+ * This class defers textdomain loading until the language is defined either in a `plugins_loaded` action
+ * or in a `wp` action (when the language is set from content on frontend).
  *
  * @since 1.2
+ * @since 3.6 Singleton removed, instantiate at your own risk!
```

Comment:
In fact it has never been a singleton. I just checked and the constructor has always been public.
```suggestion
 ```